diff --git a/device/google/wahoo/gps.conf b/device/google/wahoo/gps.conf
index 54482b19..4fe74aad 100644
--- a/device/google/wahoo/gps.conf
+++ b/device/google/wahoo/gps.conf
@@ -54,6 +54,9 @@ CAPABILITIES=0x37
 # of b1000001010 makes 3 slots available
 # and the remaining 7 slots unwritable.
 # AGPS_CERT_WRITABLE_MASK=0
+SUPL_HOST = 127.0.0.1
+SUPL_PORT = 9999
+SUPL_MODE = 0
 
 ################################
 # EXTRA SETTINGS
@@ -95,7 +98,7 @@ LPPE_UP_TECHNOLOGY = 0
 #     GPS_LOCK
 # 0: disable
 # 1: enable
-AGPS_CONFIG_INJECT = 1
+AGPS_CONFIG_INJECT = 0
 
 # AP Coarse Timestamp Uncertainty
 ##################################################
diff --git a/hardware/qcom/gps/etc/gps.conf b/hardware/qcom/gps/etc/gps.conf
index 3bec29a..0b27282 100644
--- a/hardware/qcom/gps/etc/gps.conf
+++ b/hardware/qcom/gps/etc/gps.conf
@@ -17,11 +17,11 @@ XTRA_VERSION_CHECK=0
 ERR_ESTIMATE=0
 
 #Test
-NTP_SERVER=time.gpsonextra.net
+#NTP_SERVER=time.gpsonextra.net
 #Asia
 # NTP_SERVER=asia.pool.ntp.org
 #Europe
-# NTP_SERVER=europe.pool.ntp.org
+NTP_SERVER=europe.pool.ntp.org
 #North America
 # NTP_SERVER=north-america.pool.ntp.org
 
@@ -47,12 +47,12 @@ INTERMEDIATE_POS=0
 SUPL_VER=0x10000
 
 # Emergency SUPL, 1=enable, 0=disable
-SUPL_ES=1
+SUPL_ES=0
 
 #Choose PDN for Emergency SUPL
 #1 - Use emergency PDN
 #0 - Use regular SUPL PDN for Emergency SUPL
-USE_EMERGENCY_PDN_FOR_EMERGENCY_SUPL=1
+USE_EMERGENCY_PDN_FOR_EMERGENCY_SUPL=0
 
 #SUPL_MODE is a bit mask set in config.xml per carrier by default.
 #If it is uncommented here, this value will over write the value from
@@ -60,6 +60,9 @@ USE_EMERGENCY_PDN_FOR_EMERGENCY_SUPL=1
 #MSA=0X2
 #MSB=0X1
 #SUPL_MODE=
+SUPL_MODE = 0
+SUPL_HOST = 127.0.0.1
+SUPL_PORT = 9999
 
 # GPS Capabilities bit mask
 # SCHEDULING = 0x01
